schemas/common/extensible.schema.json

```diff
@@ -225,6 +225,11 @@
                   "type": "string",
                   "format": "url",
                   "const": "https://ns.adobe.com/adcloud/"
+                },
+                "id3": {
+                  "type": "string",
+                  "format": "url",
+                  "const": "http://id3.org/ns/2.4/"
                 }
```

Comment:
In addition, we need to add id3 to the context.jsonld file here - https://github.com/adobe/xdm/blob/master/schemas/common/context.jsonld

Reply:
Done